ddspipe_core/include/ddspipe_core/configuration/DdsPipeConfiguration.hpp

```diff
@@ -121,6 +122,9 @@ struct DdsPipeConfiguration : public IConfiguration
 
     //! The type of the entity whose discovery should trigger the discovery callbacks.
     DiscoveryTrigger discovery_trigger = DiscoveryTrigger::READER;
+
+    // Logging Configuration.
```

Comment:
Please write a better comment. Comments must be descriptive, if not they are useless. An example of a better comment would be "Configuration of the CustomStdLogConsumer".